modules/ROOT/pages/clauses/merge.adoc

```diff
@@ -526,6 +526,29 @@ While there is a matching unique `Person` node with the name `'Oliver Stone'`, t
 Node already exists with label `Person` and property `name` = 'Oliver Stone'
 ----
 
+[[query-merge-using-relationship unique-constraints]]
+== Using relationship property uniqueness constraints with `MERGE`
+
+All that has been said above about node uniqueness constraints also applies to relationship uniqueness constraints.
+However, for relationship uniqueness constraints there are some additional things to consider.
+
+For example, given a relationship uniqueness constraint on `()-[:ACTED_IN(year)]-()` and a query where some or all of the nodes of the pattern are not bound, e.g., `MERGE (charlie:Person {name: 'Charlie Sheen'})-[r:ACTED_IN {year: 1987}]->(wallStreet:Movie {title: 'Wall Street'})`.
+Then by the all-or-nothing semantics of `MERGE`, the query will fail if there exists a relationship with the given `year` property but there is no match for the full pattern.
+Since no match was found for the pattern, `MERGE` will try to create the full pattern including a relationship with `{year: 1987}`, which will lead to constraint violation error.
```

Comment:
```suggestion
For example, if there exists a relationship uniqueness constraint on `()-[:ACTED_IN(year)]-()`, then the following query,  in which not all nodes of the pattern are bound, would fail:
	
[source, cypher, role=test-fail]
----
MERGE (charlie:Person {name: 'Charlie Sheen'})-[r:ACTED_IN {year: 1987}]->(wallStreet:Movie {title: 'Wall Street'})
RETURN charlie.name, type(r), wallStreet.title
----
	
This is due to the all-or-nothing semantics of `MERGE`, which causes the query to fail if there exists a relationship with the given `year` property but there is no match for the full pattern.
In this example, since no match was found for the pattern, `MERGE` will try to create the full pattern including a relationship with `{year: 1987}`, which will lead to constraint violation error.
```